jest-dynamodb-config.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+'use strict';
+
+module.exports = {
+  port: 8000,
+  tables: [{
+    BillingMode: 'PAY_PER_REQUEST',
+    TableName: 'forumTable',
+    TimeToLiveSpecification: {
+      AttributeName: 'timeToLive',
+      Enabled: true,
+    },
+    AttributeDefinitions: [{
+      AttributeName: 'threadId',
```

Comment:
The "spec" was calling this attribute "PK" as there may be other partition key than thread's.

Reply:
What spec, AWS? I did not find anything about naming the attribute PK :thinking:

Reply:
My spec ;-) 
```
forum: 

- PK (PK) "THREAD#"participant_id"#"item_id
- time (SK) time of event
- |     |     |
    | --- | --- |
    | **"type"** | **other data** |
    | "thread_opened" | { by_user_id } |
    | "attempt_started" | { attempt_id } |
    | "submission" | { attempt_id, answer_id, score?, validated? } |
    | "thread_closed" | { by_user_id } |
    | "message" | { user_id, content } |
    | "follow" | { user_id, connection_id, ttl } |
```